capability/capability_linux.go

```diff
@@ -364,19 +364,22 @@ func (c *capsV3) Apply(kind CapType) (err error) {
 	}
 
 	if kind&AMBS == AMBS {
+		err = prctl(pr_CAP_AMBIENT, pr_CAP_AMBIENT_CLEAR_ALL, 0, 0, 0)
+		if err != nil && err != syscall.EINVAL { //nolint:errorlint // Errors from syscall are bare.
+			// Ignore EINVAL as not supported on kernels before 4.3
+			return
+		}
 		for i := Cap(0); i <= last; i++ {
-			action := pr_CAP_AMBIENT_LOWER
 			if c.Get(AMBIENT, i) {
```

Comment:
nit: can we please do
```go
if ! c.Get(AMBIENT, i) {
	continue
}
err = prctl(pr_CAP_AMBIENT, pr_CAP_AMBIENT_RAISE, uintptr(i), 0, 0)
...
```
instead? This reduces indentation and improves readability a bit.